api/controllers/DeclarationController.js

```diff
@@ -27,27 +27,6 @@ module.exports = {
   },
 
   store: async function (req, res) {
```

Comment:
Not a big issue but I'm pretty sure in the future we can just delete the params to prevent linting errors from popping up.